Source/Core/oneTimeWarning.js

```diff
@@ -0,0 +1,50 @@
+/*global define*/
+define([
+           './defined',
```

Comment:
We keep the formatting of the `define` section consistent.  To fix this, just run `npm run sortRequires` and then submit the files it updates.